src/include/sof/samples/audio/kwd_nn_detect_test.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Cristina Feies <cristina.ilie@nxp.com>
+ * Author: Viorel Suman <viorel.suman@nxp.com>
+ */
+
+#ifndef __USER_KWD_NN_DETECT_TEST_H__
+#define __USER_KWD_NN_DETECT_TEST_H__
+
+#include <sof/samples/audio/kwd_nn/kwd_nn_config.h>
+
+#define KWD_NN_SILENCE         0
+#define KWD_NN_UNKNOWN         1
+#define KWD_NN_YES_KEYWORD     2
+#define KWD_NN_NO_KEYWORD      3
+
+#define N               2
+/* 990 ms of data considering frame rate 16KHz */
+#define KWD_NN_KEY_LEN         15840
+#define KWD_NN_NUM_OF_CHANNELS 1
+/* ~2 seconds of samples */
```

Comment:
Please write KWD_NN_KEY_LEN as `#define KWD_NN_KEY_LEN (990 * 16 * 1)` the preprocessor will compute this at compile time but people will understand it better at read time.

Reply:
When looking at the individual commits I can still see the 15840 value. Did you change it to the multiplication in another commit?

Reply:
It's added in commit 4 - https://github.com/thesofproject/sof/pull/3927/commits/4c653bc357c0559ad8de4d8932296efab38fc6cb#diff-643d76674a5006eaa8f2054f78d125e73b1a166e68a56e5b997a029b4fc74afdR22

Reply:
I've modified also in the original commit.